torch/nn/utils/rnn.py

```diff
@@ -234,7 +235,10 @@ def pad_sequence(sequences, batch_first=False, padding_value=0):
     # in sequences are same and fetching those from sequences[0]
     max_size = sequences[0].size()
     trailing_dims = max_size[1:]
-    max_len = max([s.size(0) for s in sequences])
+    max_len = max((s.size(0) for s in sequences))
+    if min_length:
```

Comment:
This conditional is unnecessary. Since the default is `0`, and it's an unsigned values, then `max(x, 0) == x`, so you can always apply it.